test/test_simulation_execute.jl

```diff
@@ -1,45 +1,159 @@
+                               "ED" => Stage(GenericOpProblem, template_ed, c_sys5_ed, GLPK_optimizer))
 
     sequence = SimulationSequence(order = Dict(1 => "UC", 2 => "ED"),
-                   intra_stage_chronologies = Dict(("UC"=>"ED") => Synchronize(from_steps = 24, to_executions = 1)),
-                   horizons = Dict("UC" => 24, "ED" => 12),
-                   intervals = Dict("UC" => Hour(24), "ED" => Hour(1)),
-                   feed_forward = Dict(("ED", :devices, :Generators) => SemiContinuousFF(binary_from_stage = :ON, affected_variables = [:P])),
-                   cache = Dict("ED" => [TimeStatusChange(:ON_ThermalStandard)]),
-                   ini_cond_chronology = Dict("UC" => Consecutive(), "ED" => Consecutive())
-                   )
-
-        sim = Simulation(name = "test",
-                 steps = 2,
-                 step_resolution=Hour(24),
-                 stages = stages_definition,
-                 stages_sequence = sequence,
-                 simulation_folder= file_path,
-                 verbose = true)
-
-    build!(sim)                 
+                horizons = Dict("UC" => 24, "ED" =>12),
+                intervals = Dict("UC" => Hour(24), "ED" => Hour(1)),
+                feed_forward = Dict(("ED", :devices, :Generators) => SemiContinuousFF(binary_from_stage = :ON, affected_variables = [:P])),
+                cache = Dict("ED" => [TimeStatusChange(:ON_ThermalStandard)]),
+                ini_cond_chronology = Dict("UC" => RecedingHorizon(), "ED" => RecedingHorizon())
+                )
+
+    sim = Simulation(name = "receding",
+                    steps = 2,
+                    step_resolution = Hour(24),
+                    stages = stages_definition,
+                    stages_sequence = sequence,
+                    simulation_folder= file_path,
+                    verbose = true)
+    build!(sim)
     sim_results = execute!(sim)
-    stage = [1, 2]
+    results = load_simulation_results(sim_results, "UC")
+
+    @testset "Testing to verify time gap for Receding Horizon" begin
+        names = ["UC"] # TODO why doesn't this work for ED??
+        for name in names
+            variable_list = PSI.get_variable_names(sim, name)
+            reference_1 = PSI.get_reference(sim_results, name, 1, variable_list[1])[1]
+            reference_2 = PSI.get_reference(sim_results, name, 2, variable_list[1])[1]
+            time_file_path_1 = joinpath(dirname(reference_1), "time_stamp.feather") #first line, file path
+            time_file_path_2 = joinpath(dirname(reference_2), "time_stamp.feather")
+            time_1 = convert(Dates.DateTime, Feather.read(time_file_path_1)[1,1]) # first time
+            time_2 = convert(Dates.DateTime, Feather.read(time_file_path_2)[1,1])
+            time_change = time_2 - time_1
+            interval = sim.sequence.intervals[name] # change to getter function
+            @test Dates.Hour(time_change) == Dates.Hour(interval)
+        end
+    end
+#=
```

Comment:
Can you include comments describing why these tests are commented out?

Reply:
Please uncomment the test or update them,  even if they are failing because the functionality is not implemented